BillTrack.Api/Configurations/UserInitializer.cs

```diff
@@ -0,0 +1,39 @@
+using BillTrack.Domain.Entities;
+using BillTrack.Persistence;
+
+namespace BillTrack.Api.Configurations;
+
+public static class UserInitializer
+{
+    public static WebApplication Seed(this WebApplication app)
+    {
+        using var scope = app.Services.CreateScope();
+        using var context = scope.ServiceProvider.GetRequiredService<AppDbContext>();
```

Comment:
if `AppDbContext` is registered already is there a possibility, that we could inject it via constructor instead of `WebApplication app`? Then we could not write these 2 rows and use the injected `context`.

Reply:
Injecting AppDbContext via the constructor wouldn't work here because static classes and methods don't support constructor injection. We need to manually resolve the context within the method to ensure it's properly scoped and managed by the DI container. This approach ensures everything is initialized correctly without running into lifecycle or disposal issues.